packages/veritone-react-common/src/components/OCREngineOutputView/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component, Fragment } from 'react';
+import { arrayOf, shape, number, string, func } from 'prop-types';
+import classNames from 'classnames';
+
+import EngineOutputHeader from '../EngineOutputHeader';
+import withMuiThemeProvider from '../../helpers/withMuiThemeProvider';
+import OCRObject from './OCRObject';
+import styles from './styles.scss';
+
+@withMuiThemeProvider
+class OCREngineOutputView extends Component {
+  static propTypes = {
```

Comment:
make sure to specify required props if any

Reply:
done